css/style.css

```diff
@@ -0,0 +1,81 @@
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+  padding: 0;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+}
+.text-area {
+  width: 42%;
+  height: 24%;
+  margin-bottom: 30px;
+  font-size: 23px;
+  box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.75);
+}
+.keyboard-container {
+  display: flex;
+  justify-content: space-around;
+  align-content: space-around;
+  flex-wrap: wrap;
+  width: 893px;
+  height: 320px;
+}
+.key-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 54px;
+  height: 54px;
+  border: 1px solid #0000005c;
+  border-radius: 5px;
+  box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.75);
+  cursor: pointer;
+  transition: all 70ms linear;
+}
+.key-container:hover {
+  background-color: #00000019;
+}
+.double-key-container {
+  width: 108px;
+}
+.right-arrow {
+  margin-right: 53px;
+}
+.space {
+  width: 296px;
+}
+.key-container span {
+  display: flex;
+  width: 60%;
+}
+.main-span {
+  justify-content: center;
+  font-size: 20px;
+}
+.alternative-span {
+  justify-content: flex-end;
+  font-size: 13px;
+}
+.pressed-key-container {
+  box-shadow: none;
+  background-color: #00000025;
+}
+.caps-lock-indicator-container {
+  display: flex;
+  width: 60%;
+  justify-content: flex-end;
+}
+.caps-lock-indicator {
+  width: 10px;
+  height: 5px;
+  border-radius: 4px;
+  background-color: #80808099;
```

Comment:
I think there is alpha canal here in rgba mode and you convert it to hex and this causes more than 6 symbols